presentation/src/main/java/com/playone/mobile/presentation/createPlayone/CreatePlayoneContract.kt

```diff
@@ -0,0 +1,25 @@
+package com.playone.mobile.presentation.createPlayone
+
+import com.playone.mobile.presentation.BasePresenter
+import com.playone.mobile.presentation.BaseView
+import com.playone.mobile.presentation.model.PlayoneView
+import com.playone.mobile.presentation.onBoarding.LoginPlayoneContract
+import java.util.Date
+
+interface CreatePlayoneContract {
+
+    interface View : BaseView<LoginPlayoneContract.Presenter, PlayoneView>
+
+    interface Presenter : BasePresenter {
+
+        fun create(name: String,
```

Comment:
The parameter is a class with all parameter will be awesome!

Reply:
You think it is better to make another class for paramters?

Reply:
I think it's better as possible as we can. Once the parameters are changed, we need to change many many places(!?) at the same time.
If we keep the data class `data class CreateParameters`, it might be awesome to use it as a parameter here.